assignments/function-conversion.js

```diff
@@ -4,27 +4,40 @@
 // console.log("Function was invoked!");
 // };
 // myFunction();
+let myFunction = () => console.log("Function was invoked!");
```

Comment:
Remember to always use `const` as a default first, then adjust to `let` if necessary.